.readthedocs.yaml

```diff
@@ -3,18 +3,16 @@
 version: 2
 
 build:
-  image: latest
+  os: ubuntu-22.04
+  tools:
+    python: 3
 
 formats: all
 
 python:
   version: 3.7
```

Comment:
```suggestion
```

it seems that this configuration value isn't effective anymore: https://docs.readthedocs.io/en/stable/config-file/v2.html#python

Reply:
lmao